src/ray/test/run_object_manager_tests.sh

```diff
@@ -45,7 +45,10 @@ sleep 1s
 # Run tests.
 $CORE_DIR/src/ray/object_manager/object_manager_stress_test $STORE_EXEC
 sleep 1s
-$CORE_DIR/src/ray/object_manager/object_manager_test $STORE_EXEC
+# Use timeout=1000ms for the Wait tests.
+$CORE_DIR/src/ray/object_manager/object_manager_test $STORE_EXEC 1000
+# Run tests again with inlined objects.
+$CORE_DIR/src/ray/object_manager/object_manager_test $STORE_EXEC 1000
```

Comment:
Why use the same parameters for the same test? Is `true` missing?

Reply:
Ah, thank you! I added it.